GVRf/Framework/framework/src/main/jni/objects/bounding_volume.cpp

```diff
@@ -182,6 +182,9 @@ void BoundingVolume::transform(const BoundingVolume &in_volume,
             max_corner_.z += a;
         }
     }
+    // expand with the new corners
+    expand(min_corner_);
```

Comment:
why are these two lines required? 

min_corner_ and max_corner_ are set by this function itself.